ITGuru.FourWheels/ITGuru.FourWheels.Web/Pages/Testpage.cshtml

```diff
@@ -0,0 +1,4 @@
+﻿@page
+@model ITGuru.FourWheels.Web.Pages.TestpageModel
+@{
```

Comment:
Why is there no HTML/CSS added to the site? How are you able to test this?

Reply:
We dont need HTML/CSS!